src/AzSK.ADO/Framework/BugLog/BugMetaInfoProvider.ps1

```diff
@@ -231,4 +231,31 @@ class BugMetaInfoProvider {
         return $assignee;
     }
 
+    #method to obtain sign in ID of TF scoped identities
+    hidden [string] GetAssigneeFromTFScopedIdentity($identity,$organizationName){
+        $assignee = $null;
+        #TF scoped identities with alternate email address will be in the format: a.b@microsoft.com
+        if($identity -like "*.*@microsoft.com"){
+            #check for the correct identitity corresponding to this email
+            $url="https://dev.azure.com/{0}/_apis/IdentityPicker/Identities?api-version=7.1-preview.1" -f $organizationName
+            $body = "{'query':'{0}','identityTypes':['user'],'operationScopes':['ims','source'],'properties':['DisplayName','IsMru','ScopeName','SamAccountName','Active','SubjectDescriptor','Department','JobTitle','Mail','MailNickname','PhysicalDeliveryOfficeName','SignInAddress','Surname','Guest','TelephoneNumber','Description'],'filterByEntityIds':[],'options':{'MinResults':40,'MaxResults':40}}" | ConvertFrom-Json
```

Comment:
as we are using only "signInAddress" as
$assignee = $responseObj.results[0].identities[0].signInAddress 
then why are we including these properties to return from api 'DisplayName','IsMru','ScopeName','SamAccountName','Active','SubjectDescriptor','Department','JobTitle','Mail','MailNickname','PhysicalDeliveryOfficeName','SignInAddress','Surname','Guest','TelephoneNumber','Description'